01week/data-types.js

```diff
@@ -0,0 +1,54 @@
+//Write a JavaScript program to display the current day and time.
+const date = new Date();
+ 
+console.log(date);
+
+//Write a JavaScript program to convert a number to a string.
+const num = 24;
+const n = num.toString();
+console.log(n)
+
+//Write a JavaScript program to convert a string to the number.
+const a = parseInt("7");
```

Comment:
this should be a function that takes in any string and returns the string as a number